### DISCUSSION TITLE
MODULE_NOT_FOUND error during 'npm run login'

### DISCUSSION THREAD

**Discussion by myscilab1** [mozilla/hubs#5861] [Hubs Cloud Questions]
Hello Hub users,

I am new to Hubs cloud. I have a working AWS Hubs Cloud instance and am trying to setup the custom client and seeing MODULE_NOT_FOUND error during 'npm run login'. I am on a windows machine and I am running these commands from the Git Bash window. Appreciate any help.

Thanks!

$ npm run login

> hubs@0.0.1 login
> node -r @babel/register -r esm -r ./scripts/shim scripts/login.js
..
..
..
C:\\hubs\src\utils\phoenix-utils.js:1
Error: Cannot find module '../utils/store-instance'
Require stack:
- C:\\hubs\src\utils\phoenix-utils.js
- C:\\hubs\scripts\login.js
    at Object.<anonymous> (C:\\hubs\src\utils\/phoenix-utils.js:5:1)
    at Generator.next (<anonymous>) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\Chinnu.Home_PC\\Desktop\\scilab\\hubscloud\\github\\reposnew\\hubs\\src\\utils\\phoenix-utils.js',
    'C:\\Users\\Chinnu.Home_PC\\Desktop\\scilab\\hubscloud\\github\\reposnew\\hubs\\scripts\\login.js'
  ]
}

Node.js v18.12.1

**Comment by NigelRidderhof** [comment]
I just ran into this as well. On the 17th of January 2023 a - I guess - temporary fix was given in the #hubscloud Discord channel by Pellinore. As he stated today, this problem will need a proper fix from the core team at some point, but this seemed to work for me for now. 

Hope this helps! Cheers!

**Reply by manasa15M** [reply]
Hello 
I'm also getting the same error "module not found" while running npm run login
Can you please help me with the fix that you have implemented
